# Set a parameter value
#
# Set either the integer_value, unsigned_value, or
# float_value field depending on the the parameter type

# see enum MAV_PARAM_TYPE
uint8 MAV_PARAM_TYPE_UINT8  = 1
uint8 MAV_PARAM_TYPE_INT8   = 2
uint8 MAV_PARAM_TYPE_UINT16 = 3
uint8 MAV_PARAM_TYPE_INT16  = 4
uint8 MAV_PARAM_TYPE_UINT32 = 5
uint8 MAV_PARAM_TYPE_INT32  = 6
uint8 MAV_PARAM_TYPE_REAL32 = 9

string param_id
uint8 param_type

int32 integer_value
uint32 unsigned_value
float32 float_value
---
